server/init_express.coffee

```diff
@@ -4,30 +4,34 @@ _ = __.require('builders', 'utils')
 express = require 'express'
 { env, port, host, name } = CONFIG
 
+console.time 'middlewares'
 middlewares = require './middlewares/middlewares'
 middlewaresList = middlewares.common.concat (middlewares[CONFIG.env] or [])
 
 middlewareErrorHandler = require './middlewares/error_handler'
+console.timeEnd 'middlewares'
 
+console.time 'routes'
 routes = require './controllers/routes'
+console.timeEnd 'routes'
```

Comment:
is there a need to leave those benchmark tool displaying at every start up ?

Reply:
removed